### PR TITLE
chore(docs): Update uninstall instructions

### PR DESCRIPTION
**Pull request by Bharat123rox**
Close openebs/openebs#3121 (cleanup in `uninstall.md`)

### CODE REVIEW

File: docs/uninstall.md [hunk 1]
```diff
@@ -107,7 +107,8 @@ The recommended steps to uninstall the OpenEBS cluster gracefully is as follows.
   ```
   
   If present, remove the finalizer entry from the corresponding BD and then delete it.
-
+  
+  After removing a pvc, we can still find a directory `/var/openebs/` having some of the volume sub-directory to store metadata info related to volume, for example `shared-pvc-69adec76-665e-46bc-b957-2b2f58338429-target` with no content. This can be removed manually once you successfully uninstall the OpenEBS cluster.
```
Comment (ranjithwingrider) [inline docs/uninstall.md:111]: Thanks @Bharat123rox  for the changes. Can you please make the following suggested changes?
1. pvc -> PVC
2.  we can still find a directory `/var/openebs/` -> you may find a directory `/var/openebs/` on the nodes where the replica was created
3. related to volume -> related to the volume
4. removed manually -> removed manually using `rm -rf` command

**Review by ranjithwingrider — CHANGES_REQUESTED** [review]
Please make the suggested change. Thanks.

File: docs/uninstall.md (hunk 1 above)
Comment (Bharat123rox) [reply docs/uninstall.md]: @ranjithwingrider Updated

**Review by ranjithwingrider — APPROVED** [review]
Looks fine...

**Review by kmova — APPROVED** [review]

**Pull request merged**